staging/src/k8s.io/client-go/util/workqueue/delaying_queue_test.go

```diff
@@ -214,6 +215,25 @@ func TestCopyShifting(t *testing.T) {
 	}
 }
 
+func BenchmarkDelayingQueue_AddAfter(b *testing.B) {
+	r := rand.New(rand.NewSource(time.Now().Unix()))
+
+	fakeClock := clock.NewFakeClock(time.Now())
+	q := newDelayingQueue(fakeClock, "")
+
+	// Add items
+	for n := 0; n < b.N; n++ {
+		data := fmt.Sprintf("%d", n)
+		q.AddAfter(data, time.Duration(r.Int63n(int64(10*time.Minute))))
+	}
+
+	// Exercise item removal as well
+	fakeClock.Step(10 * time.Minute)
+	for q.Len() > 0 {
+		_, _ = q.Get()
+	}
+}
```

Comment:
thanks, curious what the affect on the benchmark was (mind updating the description)?